frontend/src/app/account-activation/account-activation.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { AccountActivationRequestModel } from 'app/auth.models';
+import { Observable } from 'rxjs/Observable';
+import { AuthService } from '../auth.service';
+import { Router, ActivatedRoute, Params } from '@angular/router';
+
+@Component({
+  selector: 'volontulo-account-activation',
+  templateUrl: './account-activation.component.html',
+  styleUrls: ['./account-activation.component.css']
+})
+export class AccountActivationComponent implements OnInit {
+  accountActivationRequestModel: AccountActivationRequestModel = {
+    uuid: '',
+  };
+  activationSuccessful: boolean;
+
+  constructor(private authService: AuthService, private activatedRoute: ActivatedRoute
+  ) {
+    this.activationSuccessful = false;
+  }
+
+  activate(): void {
+    this.authService.account_activation(this.accountActivationRequestModel.uuid)
+      .subscribe(rsp => {
```

Comment:
Shouldn't it be `map` here instead of `subscribe` and intead `this.activationSuccessful = true` we could return result as `Observable.of(result)` and subscribe to it inside a template using `| async`?